StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsNetworkedAccountsResponse.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  FinancialConnectionsNetworkedAccountsResponse.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 6/16/23.
+//
+
+import Foundation
+
+struct FinancialConnectionsNetworkedAccountsResponse: Decodable {
+    let data: [FinancialConnectionsPartnerAccount]
+    let display: Display?
+    let nextPaneOnAddAccount: FinancialConnectionsSessionManifest.NextPane?
+
+    struct Display: Decodable {
+        let text: Text?
```

Comment:
can we directly declare `FinancialConnectionsNetworkingAccountPicker` here instead of wrapping inside `Text`?

Reply:
If I understood the request correctly, this is a representation of a backend model (we make a network call that returns a JSON, and this JSON is automatically turned into a model). As a result, we can't change its structure (its structure was decided as part of a different process from the server team).